diff -Naur Sonarr-4.0.13.2932.old/src/NzbDrone.Core/Sonarr.Core.csproj Sonarr-4.0.13.2932/src/NzbDrone.Core/Sonarr.Core.csproj
--- Sonarr-4.0.13.2932.old/src/NzbDrone.Core/Sonarr.Core.csproj	2025-03-11 09:31:54.931495658 +0100
+++ Sonarr-4.0.13.2932/src/NzbDrone.Core/Sonarr.Core.csproj	2025-03-11 09:32:25.648838407 +0100
@@ -20,7 +20,7 @@
     <PackageReference Include="Servarr.FluentMigrator.Runner.SQLite" Version="3.3.2.9" />
     <PackageReference Include="Servarr.FluentMigrator.Runner.Postgres" Version="3.3.2.9" />
     <PackageReference Include="FluentValidation" Version="9.5.4" />
-    <PackageReference Include="SixLabors.ImageSharp" Version="3.1.6" />
+    <PackageReference Include="SixLabors.ImageSharp" Version="3.1.7" />
     <PackageReference Include="Newtonsoft.Json" Version="13.0.3" />
     <PackageReference Include="NLog" Version="5.3.4" />
     <PackageReference Include="MonoTorrent" Version="2.0.7" />
