diff -Naur Sonarr-4.0.15.2941.old/src/NzbDrone.Core/Sonarr.Core.csproj Sonarr-4.0.15.2941/src/NzbDrone.Core/Sonarr.Core.csproj
--- Sonarr-4.0.15.2941.old/src/NzbDrone.Core/Sonarr.Core.csproj	2025-09-06 10:46:52.622983762 +0200
+++ Sonarr-4.0.15.2941/src/NzbDrone.Core/Sonarr.Core.csproj	2025-09-06 10:47:06.518644535 +0200
@@ -20,7 +20,7 @@
     <PackageReference Include="Servarr.FluentMigrator.Runner.SQLite" Version="3.3.2.9" />
     <PackageReference Include="Servarr.FluentMigrator.Runner.Postgres" Version="3.3.2.9" />
     <PackageReference Include="FluentValidation" Version="9.5.4" />
-    <PackageReference Include="SixLabors.ImageSharp" Version="3.1.7" />
+    <PackageReference Include="SixLabors.ImageSharp" Version="3.1.11" />
     <PackageReference Include="Newtonsoft.Json" Version="13.0.3" />
     <PackageReference Include="NLog" Version="5.3.4" />
     <PackageReference Include="MonoTorrent" Version="2.0.7" />
